go/runtime/client/client.go

```diff
@@ -428,6 +449,11 @@ func New(
 	runtimeRegistry runtimeRegistry.Registry,
 	p2p *p2p.P2P,
 ) (api.RuntimeClient, error) {
+	maxTransactionAge := viper.GetInt64(CfgMaxTransactionAge)
+	if maxTransactionAge < minMaxTransactionAge && !cmdFlags.DebugDontBlameOasis() {
+		return nil, fmt.Errorf("max transaction age to low: %d", maxTransactionAge)
```

Comment:
```suggestion
		return nil, fmt.Errorf("max transaction age too low: %d", maxTransactionAge)
```
(also maybe mention what the minimum is)

Reply:
:+1: